test/playwright.spec.js

```diff
@@ -172,6 +172,9 @@ module.exports.describe = ({testRunner, product, playwrightPath}) => {
       testRunner.loadTests(require('./geolocation.spec.js'), testOptions);
       testRunner.loadTests(require('./workers.spec.js'), testOptions);
 
+      if (WEBKIT)
```

Comment:
Why not everywhere? It makes sense to smoke test that wasm works in all our browsers.

Reply:
SG!